pkg/kube_resource/generator/generator.go

```diff
@@ -150,7 +150,12 @@ func buildSwagger(crd *apiextensions.CustomResourceDefinition) (*spec.Swagger, e
 		if err != nil {
 			return nil, err
 		}
-		version := crd.Spec.Version
+		var version string
+		if len(crd.Spec.Versions) >= 0 {
```

Comment:
Would you pls add more comments and unit test cases? Or you can ref the issue directly.

Reply:
the issue is already refered in the PR content